backend/app/sql/summary_stats_pop_totals.sql

```diff
@@ -0,0 +1,40 @@
+DROP FUNCTION IF EXISTS get_summary_stats_pop_totals;
```

Comment:
H: This function isn't being used now, no? We can delete in favor of using the either `get_summary_p1_totals` or `get_summary_p4_totals` (and eventually others...)?